apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/ClientCertificateUtils.java

```diff
@@ -0,0 +1,71 @@
+package gov.cms.bfd.server.war.commons;
+
+import jakarta.servlet.http.HttpServletRequest;
+import java.math.BigInteger;
+import java.security.cert.X509Certificate;
+import javax.security.auth.x500.X500Principal;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/***
+ * test.
+ */
+public class ClientCertificateUtils {
```

Comment:
Most of this was moved here from `RequestResponseMdcFilter` so it can be reused.